fullscreen.bs

```diff
@@ -285,15 +286,15 @@ are:
     </thead>
     <tbody>
       <tr>
-        <td>"<code>hide</code>"</td>
+        <td>"<dfn enum-value for="FullscreenNavigationUI"><code>hide</code></dfn>"</td>
```

Comment:
I think `{{FullscreenNavigationUI/hide}}` also works, no?

Reply:
The intention is to define it here; that would leave the definition in the IDL and reference it.